workers/unity/Assets/Fps/Scripts/UI/Controllers/Testers/DeploymentListScreenTester.cs

```diff
@@ -0,0 +1,51 @@
+﻿using UnityEngine;
+
+namespace Fps
+{
+    [RequireComponent(typeof(DeploymentListScreenController))]
+    public class DeploymentListScreenTester : MonoBehaviour
+    {
+        public bool Test;
+        public int NumResultsToGenerate;
+
+        private void Update()
+        {
+            if (!Test)
+            {
+                return;
+            }
+
+            Test = false;
+            RunTest();
+        }
+
+        private void OnEnable()
+        {
+            Test = false;
+            Invoke(nameof(RunTest), 1);
```

Comment:
Why is this delayed?

Reply:
If deployments are added to the table in `OnEnable()`, there's a chance `Awake()` hasn't yet been called on the `Table` object, which is where the new entry template reference is established. If that happens, you get a null ref exception.

It's not great, is it? What's the alternative, to have a getter on `Table` that auto-gathers the template ref if it doesn't exist yet?

Reply:
Reading my comment again it sounds like I might be being sassy...that wasn't my intention! I'm genuinely looking for a better solution to this and wondering if what I suggested would be acceptable.